pulp_rpm/tests/functional/utils.py

```diff
@@ -79,83 +79,16 @@ def get_rpm_package_paths(repo):
     ]
 
 
-def populate_pulp(cfg, url=RPM_SIGNED_FIXTURE_URL):
```

Comment:
is it not needed anymore?

Reply:
same here, anywhere used

Reply:
maybe it is used on jenkins,
@rochacbruno could you confirm if QE uses this?

Reply:
It is ok to remove it @pavelpicka @fabricio-aguiar looks like it is a duplicate https://github.com/pulp/pulpcore/blob/master/pulpcore/tests/functional/api/using_plugin/utils.py#L30-L55

Reply:
It's not a "duplicate" since the one linked is a clone of the one from the File plugin, not the RPM plugin. 

With that said, we should try to consolidate utility functions like this if we can because they're copied and pasted everywhere with minor changes.